nginx/etc/nginx/nginx-ssl.conf.ctmpl

```diff
@@ -0,0 +1,95 @@
+user  nginx;
+worker_processes  1;
```

Comment:
This one can be deleted/merged now that https://github.com/autopilotpattern/nginx/pull/32 is upstream.